cloud/db_cloud_impl.cc

```diff
@@ -146,8 +146,9 @@ Status DBCloud::Open(const Options& opt, const std::string& local_dbname,
     }
   }
   if (new_db) {
-    if (read_only) {
-      return Status::NotFound("CLOUDMANIFEST not found and read_only is set.");
+    if (!options.create_if_missing) {
```

Comment:
If `read_only` is true, `create_if_missing` is false

Reply:
we should not get rid of the "If (read_only)"

if (read_only || !options.create_if_missing) {
  return error;
}